File: seed/java-sdk/exhaustive/local-files/src/main/java/com/snippets/Example45.java
package com.snippets;

import com.fern.sdk.SeedApiClient;
import com.fern.sdk.types.TypesObjectWithRequiredField;

public class Example45 {
    public static void main(String[] args) {
        SeedApiClient client = SeedApiClient
            .builder()
            .token("<token>")
            .url("https://api.fern.com")
            .build();

        client.endpoints().object().getAndReturnWithRequiredField(
            TypesObjectWithRequiredField
                .builder()
                .string("string")
                .build()
        );
    }
}

File: seed/csharp-sdk/exhaustive/no-root-namespace-for-core-classes/Snippets/Example45.cs
using SeedApi;
using SeedApi.Core;

public partial class Examples
{
    public async Task Example45() {
        var client = new SeedApiClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Endpoints.Object.GetAndReturnWithRequiredFieldAsync(
            new TypesObjectWithRequiredField {
                String = "string"
            }
        );
    }

}
